test/Makefile

```diff
@@ -57,6 +57,10 @@ net2cog-image: Dockerfile net2cog/environment.yaml
 	docker build -t ghcr.io/nasa/regression-tests-net2cog:latest -f ./Dockerfile \
 	--build-arg notebook=net2cog_Regression.ipynb --build-arg sub_dir=net2cog .
 
+sambah-image: Dockerfile sambah/environment.yaml
+	docker build -t ghcr.io/nasa/regression-tests-sambah:latest -f ./Dockerfile \
+	--build-arg notebook=SAMBAH_Regression.ipynb --build-arg sub_dir=sambah .
```

Comment:
This is the other place you need to include the shared utils. I am open to ideas on how to make this clearer/easier. [flamingbear/harmony-regression-tests@`7969125` (#1)](https://github.com/flamingbear/harmony-regression-tests/pull/1/commits/79691251716f1ad7be8c7aca18593252fd8b2051)